src/Resources/ResourceManager/Implementation/Policy/SetAzurePolicyAssignment.cs

```diff
@@ -167,6 +182,7 @@ private JToken GetResource(string resourceId, string apiVersion)
                     NotScopes = this.NotScope ?? resource.Properties["NotScopes"]?.ToString().Split(','),
                     PolicyDefinitionId = resource.Properties["policyDefinitionId"].ToString(),
                     Metadata = string.IsNullOrEmpty(this.Metadata) ? null : JObject.Parse(metaDataJson),
+                    EnforcementMode = this.EnforcementMode ?? existingMode,
```

Comment:
I think existingMode will enter this code as null in the case of updating an existing policy that doesn't have enforcement mode. Do we want to send null to the backend in this case? I assume the backed will do the right thing, but I feel like it might be better to explicitly assign Default here. What do you think?

Reply:
GetResource on policy assignment uses latest API version that always returns this property; Default if not set originally.